async/async.go

```diff
@@ -79,7 +83,7 @@ func ExecuteWithContext(fn Task, c echo.Context, poolName ...string) {
 
 		// IMPORTANT - Set the sentry hub key into the context so that `SentryCaptureException` and `SentryCaptureMessage`
 		// can pull the right hub and send the exception message to sentry.
-		if bean.BeanConfig.Sentry.On {
+		if viper.GetBool("sentry.on") {
```

Comment:
@zhongdalu-trechina why again call `viper` GetBool func because `bean.BeanConfig.Sentry.On` this is faster than getting the value from `GetBool`

Reply:
> @zhongdalu-trechina why again call `viper` GetBool func because `bean.BeanConfig.Sentry.On` this is faster than getting the value from `GetBool`. is it because of circular dependency issue?

yeah, just want to make async look more independent. It does get slower.

Reply:
done